rs/ic_os/network/src/systemd.rs

```diff
@@ -74,6 +75,55 @@ pub fn restart_systemd_networkd() {
     // Explicitly don't care about return code status...
 }
 
+// TODO(NODE-1466): Consolidate generate_systemd_config_files_new_config and generate_and_write_systemd_files
+pub fn generate_systemd_config_files_new_config(
+    output_directory: &Path,
+    ipv6_config: &DeterministicIpv6Config,
+    generated_mac: Option<&MacAddr6>,
+    ipv6_address: &Ipv6Addr,
+) -> Result<()> {
+    let mut interfaces = get_interfaces()?;
+    interfaces.sort_by_key(|v| v.speed_mbps);
```

Comment:
You can sort in reversed order by `.sort_by_key(|v| Reverse(v.speed_mbps))`

Reply:
Nice! Fixed! Thank you!